src/interfaces/analysis-result.interface.ts

```diff
@@ -18,7 +18,15 @@ interface CommitChangeLine {
   lineChange: 'removed' | 'added' | 'none';
 }
 
-interface ExampleCommitFix {
+export interface RuleProperties {
+  tags: string[];
+  exampleCommitFixes?: ExampleCommitFix[];
+  exampleCommitDescriptions?: string[];
+  precision: string;
+  cwe?: string[];
+}
+
+export interface ExampleCommitFix {
```

Comment:
Required: I think you no longer need to export this.

Reply:
👍